scripts/lib/node_cache.py

```diff
@@ -1,9 +1,9 @@
 
 import os
 import hashlib
+import typing
 
-if False:
-    from typing import Optional, List, IO, Text, Tuple, Any
+from typing import Optional, List, IO, Text, Tuple, Any
```

Comment:
I think that error might have something to do your development environment setup, because I reverted the changes above and the mypy tests still ran fine. I think you should try reverting these changes and check out travis (CI) reacts.

Reply:
Yeah any file that is called at the beginning, during `./tools/provision` session shouldn't have the Python 3 annotation syntax enabled.